golangDay1/Day2/exercise/solution2.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+var total_rating int =0;
+func getRating(student int,wg *sync.WaitGroup) {
+	defer wg.Done()
+	//x:=rand.Intn(10)
+	time.Sleep(1*time.Second)
+	rating:= rand.Intn(10)
+	total_rating+=rating
```

Comment:
multiple go routines can try to add rating to total_rating at the same time. How are you going to handle that ?